internal/security/fileioperformer/fileioperformer.go

```diff
@@ -0,0 +1,51 @@
+//
```

Comment:
Request that this package/file name combination be reconsidered. Having a file.go that is named the same as its parent package is an example of "stutter".

See the "Avoid Stutter" sub-section in the following blog post.
https://blog.golang.org/package-names

Reply:
For the moment, to resolve the immediate issue, I renamed it to `methods.go` since that is the class that implements the methods of the interface.  This is an easy change that doesn't affect the imports of all the other packages that consume it.  Does this resolve the issue or do you feel strongly about renaming the package to "file"?

Reply:
@bnevis-i That was just a suggestion. If the naming duplication is taken care of, I'm good.